docs/pages/serving_a_federation.mdx

```diff
@@ -140,14 +140,28 @@ Then pass the file location to `Director.MaxMindKeyFile`.
 A Pelican director needs to talk to the registry to get the public keys for namespaces. You will need to set `Federation.RegistryUrl` to the public endpoint of your registry
 for your Pelican director to work. An example value looks like `https://your-registry.org:9999`
 
-### Director as a Federation (optional)
+### Director as a Federation
 
-For Pelican origins and caches, they need to know where the *federation* is so that they can
-attempt to join it. This can be done by either manually passing configuration values or by requesting the
-`Federation.DiscoveryUrl` endpoint, which contains various URLs to a federation's registry and director.
-You may supply your own endpoint to provide such information or the Pelican director can do it for you.
+For Pelican origin, cache, and client, they need to know where the *federation* is so that they can
+attempt to join it or a access a file from it. This can be done by either manually passing configuration values (e.g. `Federation.DirectorUrl`, `Federation.RegistryUrl`, etc) or by setting the
+`Federation.DiscoveryUrl` and Pelican will fetch the metadata from the endpoint automatically. `Federation.DiscoveryUrl` contains various URLs to a federation's central services including registry and director.
 
-This can be done by setting `Federation.DiscoveryUrl` of your *origins* and *caches* to the public url of your director, i.e. `https://<your-director-hostname>:<director-web-port>`. Make sure your director is up and running before starting your origins and caches.
+By default, Pelican director hosts the federation discovery endpoint. Therefore, for origins and caches in your federation, as well as Pelican clients that want to access files in your federation, their `Federation.DiscoveryUrl` value can be set to the director's public URL, i.e. `https://<your-director-hostname>:<director-web-port>` or `https://<your-director-hostname>` if the port number is simply `443`.
+
+Make sure your director is up and running before starting your origins and caches.
+
+### Set Support Contact Information (Recommended)
+> `Director.SupportContact` is only available for Pelican >=7.7.0
+
+Since the director is the entrypoint of your federation, and client users uses director's public URL as the `Federation.DiscoveryUrl`, it is convenient for users to look up information related to a federation or seek help at the director's web UI. Therefore, we recommend director admin to setup contact information at the director by `Director.SupportContactEmail` and `Director.SupportContactUrl` configuration parameters. If set, the information will be displayed in the director's web UI. See [Configuration](./parameters.mdx#Director-SupportContact) page for details.
```

Comment:
The director is your federation's entrypoint, and it is where other services in the federation can learn about each other. Because of this, it is also used to host contact information for federation administrators. Setting values for the configuration options `Director.SupportContactEmail` and `Director.SupportContactUrl` causes the director to display the information through its web UI so that users know who/how to contact you for support. See ... for more details.